media-plugins/vdr-streamdev/vdr-streamdev-0.6.3.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vdr-plugin-2
+
+DESCRIPTION="VDR Plugin: Client/Server and http streaming plugin"
+HOMEPAGE="https://projects.vdr-developer.org/projects/plg-streamdev https://github.com/vdr-projects/vdr-plugin-streamdev"
+SRC_URI="https://github.com/vdr-projects/vdr-plugin-streamdev/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/vdr-plugin-streamdev-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="client +server"
+REQUIRED_USE="|| ( client server )"
+
+BDEPEND="acct-user/vdr"
+DEPEND=">=media-video/vdr-2.3"
+RDEPEND="${DEPEND}"
+
+QA_FLAGS_IGNORED="
```

Comment:
Is this stuff prebuilt or not? Why do we need to set QA_FLAGS_IGNORED?

Reply:
All packages media-plugins/vdr-* are build with the CFLAGS taken from the BDEPENDed media-video/vdr. This may result into a QA message that this package is ignoring the current CFLAGS.
Copying the CFLAGS from main vdr package is necessary to asssure that the plugins (libraries) match exactly.